src/content/en/2020/cms.md

```diff
@@ -23,3 +22,364 @@ published: 2020-12-09T00:00:00.000Z
+
+The deviation between our analysis and W3Techs’ analysis can be explained by a difference in research methodologies. You can read more about ours on the [Methodology](./methodology) page.
+
+Our research identified 222 individual CMSs, with these ranging from a single install to millions on a single CMS.
+
+Some of them are open source (e.g. WordPress, Joomla, others) and some of them are proprietary (e.g. Wix, Squarespace, others). As we’ll discuss later, the top 3 CMSs by adoption share are all open source, but proprietary platforms have seen large increases in adoption share this year. Some CMS platforms can be used on "free" hosted or self-hosted plans, and there are also options for using these platforms on higher-tiered plans even at the enterprise level.
+
+The CMS space as a whole is a complex, federated universe of CMS ecosystems, all separated and at the same time intertwined. Our research shows CMSs are only getting more important. The minimum of 5% increase in adoption of CMSs shows that in a year when COVID-19 has created immense uncertainty, solid CMS platforms have provided some stability. As we discussed last year, these platforms play a key role for us to succeed in our collective quest for an evergreen, healthy, and vibrant web. This has become more true since, and we expect it to continue to be the case going forward.
+
+
+## Which are the most common CMS platforms, across desktop and mobile sites?
+
+Our analysis counted 222 separate CMSs. While this is a high count, 204 (92%) of these have an adoption share of 0.01% or lower. This leaves only 13 CMSs with an adoption share of between 0.1 and 1%, and 4 with a share of between 1 and 2%, and 1 with a share over this.
+
+The 1 CMS with a share over 2% is WordPress, which has a 31% usage share. This is over 15 times the share of the next most popular CMS, Joomla:
+
+{{ figure_markup(
+  image="cms-adoption-share-for-top-5-cmss.png",
+  caption="CMS adoption share for top 5 CMSs.",
+  description="Bar chart showing Wordpress dominating with 28.91% share in 2019, growing to 31.04% in 2020. This is followed by 4 other CMSs with less than 2.5% share each: Joomla (2.24% for 2019 and 2.05% for 2020), Drupal (2.21% and 1.98% respectively), Wix (0.91% and 1.28%) and Squarespace (0.76% and 0.97%). Dupal and Joomla have dropped in usage in 2020 while the rest have increased.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=1429803707&format=interactive",
+  sheets_gid="1594044364",
+  sql_file="top_cmss_yoy_all_clients.sql"
+  )
+}}
+
+Joomla and Drupal have lost 0.18% and 0.23% adoption share respectively, whilst Wix and Squarespace have gained 0.38% and 0.21% adoption share respectively. WordPress has gained 2.14% adoption share in the last year, which is a larger increase than the total share for Joomla, the next most popular CMS.
```

Comment:
the text doesn't reference the % change, so I'm happy with this

Reply:
> Joomla and Drupal have lost 0.18% and 0.23% adoption share respectively

IIUC you're referring to the change in adoption rates YoY. I think it's more accurate to either say:

> Joomla and Drupal have lost 0.18 percentage points and 0.23 percentage points adoption share respectively

Or:

> Joomla and Drupal have lost 8% and 10% adoption share respectively

I think this is an important distinction because we want to give the right impression whether these were big or small changes. Changing by -0.2 can be significant when it was only 2% to begin with.

Reply:
ok sure, done!